.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2023 National Instruments
+# Copyright (c) 2025 National Instruments
```

Comment:
@jattasNI I pushed some significant updates so reset you to make the last sanity pass and then free to release

Offline asked about validation of our playwright packages which have extremely old dependencies and I'm concerned about the eslint flat config support

Reply:
I'm here. I'm here sheesh :roll_eyes: